WordPress/Classes/Utility/BuildInformation/FeatureFlag.swift

```diff
@@ -73,8 +70,6 @@ enum FeatureFlag: Int, CaseIterable, OverrideableFlag {
     /// This key must match the server-side one for remote feature flagging
     var remoteKey: String? {
         switch self {
-            case .unifiedAuth:
-                return "wordpress_ios_unified_login_and_signup"
```

Comment:
@jkmassel - I don't know if this matters server-side, but just letting you know I'm removing the app feature flag.